libprecompiled/CRUDPrecompiled.cpp

```diff
@@ -77,3 +168,84 @@ bytes CRUDPrecompiled::call(ExecutiveContext::Ptr context, bytesConstRef param,
     }
     return out;
 }
+
+void CRUDPrecompiled::parseCondition(
+    const std::string& conditionStr, const Table::Ptr& table, Condition::Ptr& condition)
```

Comment:
`const Table::Ptr& table` this param is uncessary.

Reply:
removed it